src/arlo_plugin/camera.py

```diff
@@ -465,18 +446,17 @@ async def getSettings(self) -> List[Setting]:
                 "type": "boolean",
             }
         )
-        if not self.use_sip_webrtc_streaming:
-            result.append(
-                {
-                    "group": "General",
-                    "key": "disable_eager_streams",
-                    "title": "Disable Eager Streams",
-                    "value": self.disable_eager_streams,
-                    "description": "If eager streams are disabled, Scrypted will wait for Arlo Cloud to report that " + \
-                                "the camera stream has started before passing the stream URL to downstream consumers.",
-                    "type": "boolean",
-                }
-            )
+        result.append(
+            {
+                "group": "General",
+                "key": "disable_eager_streams",
+                "title": "Disable Eager Streams",
+                "value": self.disable_eager_streams,
+                "description": "If eager streams are disabled, Scrypted will wait for Arlo Cloud to report that " + \
+                               "the camera stream has started before passing the stream URL to downstream consumers.",
```

Comment:
perhaps add here that this is only relevant to RTSP/DASH streams

Reply:
updated to say Disable Eager Streams for RTSP/DASH